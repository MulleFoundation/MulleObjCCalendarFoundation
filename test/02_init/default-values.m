#ifdef __MULLE_OBJC__
# import <MulleObjCCalendarFoundation/MulleObjCCalendarFoundation.h>
#else
# import <Foundation/Foundation.h>
#endif


int   main( void)
{
   NSCalendar   *calendar;

   calendar = [NSCalendar calendarWithIdentifier:NSGregorianCalendar];

   printf( "firstWeekday           : %ld\n", (long) [calendar firstWeekday]);
   printf( "minimumDaysInFirstWeek : %ld\n", (long) [calendar minimumDaysInFirstWeek]);
#if 0
   printf( "timeZone               : \"%s\"\n",  [calendar timeZone] ? [[[calendar timeZone] abbreviation] UTF8String] : "nil");
   printf( "locale                 : \"%s\"\n",  [calendar locale] ? [[[calendar locale] localeIdentifier] UTF8String] : "nil");
#endif
   return( 0);
}