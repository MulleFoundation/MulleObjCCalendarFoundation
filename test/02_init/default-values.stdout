firstWeekday           : 1
minimumDaysInFirstWeek : 1
